src/dswx_sar/stage_pekel_water.py

```diff
@@ -0,0 +1,539 @@
+#!/usr/bin/env python3
+
+# Pekel's Surface Water Map staging
+# https://global-surface-water.appspot.com/download
+# Pekel, JF., Cottam, A., Gorelick, N. et al. 
+# High-resolution mapping of global surface water and its long-term changes. 
+# Nature 540, 418–422 (2016). https://doi.org/10.1038/nature20584
+
+import argparse
+import os
+import backoff
+import logging
+
+# import boto3
+import urllib.request
+
+import numpy as np
+from osgeo import gdal, osr
+from shapely.geometry import LinearRing, Polygon, box
+import shapely.ops
+import shapely.wkt
+
+
+logger = logging.getLogger('stage peke water map')
+
+# Enable exceptions
+gdal.UseExceptions()
+
+# [TODO] need to update the bucket
+"""Name of the default S3 bucket containing the full JRC Pikel's water map to crop from.
+"""
+S3_REF_WATER_BUCKET = "opera-reference-water"
+PEKEL_GOOGLE_SOURCE = "http://storage.googleapis.com/global-surface-water/downloads2021/"
+
+EARTH_APPROX_CIRCUMFERENCE = 40075017.
+EARTH_RADIUS = EARTH_APPROX_CIRCUMFERENCE / (2 * np.pi)
+
+
+def get_parser():
+    """Returns the command line parser for stage_pekel_water.py"""
+    parser = argparse.ArgumentParser(
+        description="Stage and verify reference water map for processing.",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter
+    )
+    parser.add_argument('-p', '--product', type=str, action='store',
+                        help='Input reference RTC HDF5 product')
```

Comment:
I could be wrong, but it seems the function has changed to take in RTC GEOTIFF product to compute the polygon. Can you check?